### ISSUE TITLE
Accuracy drop from YoloX vs V4

### ISSUE THREAD

**Issue by abhigoku10** [Megvii-BaseDetection/YOLOX#884]
Hi thanks for open-sourcing the code base i was evaluating the model for my use case and when i compared with yolov4 i am getting a drop in the number of detections please find the details below 
Model : yolox -L 
Inputsize: 608
Threshold: 0.5

For yolov4
![image](https://user-images.githubusercontent.com/22094053/140682005-c4f93db5-1e48-444c-9853-fc9f963557b9.png)

For yolox
![image](https://user-images.githubusercontent.com/22094053/140682025-8f2ac318-c0f1-4244-a734-77ecf091284e.png)

Please share your thoughts

**Comment by Joker316701882** [comment]
@abhigoku10 How about reducing the threshold for YOLOX? It should be easy to detect those human bodies on the second floor.

**Comment by abhigoku10** [comment]
@Joker316701882  yes by reducing the threshold i am abl to detect but in comparision with yolov4 the accuracy is always dropping irrespective of any threshold , wanted to kwn the reason behind it
